tenacity/retry.py

```diff
@@ -64,14 +64,26 @@ def __call__(self, attempt):
 
 
 class retry_if_exception_type(retry_if_exception):
-    """Retries if an exception has been raised of a certain type."""
+    """Retries if an exception has been raised of one or more types."""
 
     def __init__(self, exception_types=Exception):
         self.exception_types = exception_types
         super(retry_if_exception_type, self).__init__(
             lambda e: isinstance(e, exception_types))
 
 
+class retry_until_exception_type(retry_if_exception):
```

Comment:
It's generally admitted that
* `if not` == `unless`
* `while not` == `until`

Since  it's basically the opposite of `retry_if_exception_type` I'd suggest renaming to `retry_unless_exception_type`.